controllers/argocd/deployment.go

```diff
@@ -1171,6 +1199,11 @@ func (r *ReconcileArgoCD) reconcileRepoDeployment(cr *argoproj.ArgoCD, useTLSFor
 		return nil // Deployment found with nothing to do, move along...
 	}
 
+	if !cr.Spec.Repo.IsEnabled() {
+		log.Info("ArgoCD Repo disabled. Skipping starting ArgoCD Repo.")
```

Comment:
can we call this repo-server instead of "repo"?

Reply:
Will update. Thanks.